Update log-analytics-alerts-actions.md

rbormann opened the pull request:
Hello,

I'd like to suggest a very small update to this post so that adds the SIGNL4 mobile alerting app as a additional example for webhook Alert actions. It does not need additional back-end setup in the target system and the OMS alerts appear instantly on the mobile device:
https://www.signl4.com/blog/microsoft-oms-alerts-staff-duty/

Best regards,
René

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rbormann committed:
Update log-analytics-alerts-actions.md

Hello,

I'd like to suggest a very small update to this post so that adds the SIGNL4 mobile alerting app as a additional example for webhook Alert actions. It does not need additional back-end setup in the target system and the OMS alerts appear instantly on the mobile device:
https://www.signl4.com/blog/microsoft-oms-alerts-staff-duty/

Best regards,
René
